src/annotations/core/AnnotationArea.cpp

```diff
@@ -277,7 +283,7 @@ void AnnotationArea::finishDrawingItem(const QPointF &pos)
 		mCurrentItem->finish();
 
 		if (mCurrentItem->requiresSelectionAfterCreation()) {
-			mItemModifier->handleSelectionAt(pos, mItems, false);
+			mItemModifier->handleSelectionAt(mCurrentItem->boundingRect().topLeft(), mItems, false);
```

Comment:
Isn't this the same as position()?

Reply:
`pos` is the position where the drawing finished (it can be outside the `sceneRect`), we want to use the current item's bounding rect instead, because the item is clipped to the `sceneRect`.

Reply:
I meant the `AbstractAnnotationItem::position()`, it does basically the same:
![image](https://user-images.githubusercontent.com/12796236/109946986-1fe85780-7cd9-11eb-9419-8c9005c471ab.png)

I can fix that when I merge the fix.

Reply:
Oh yes, that's correct.